.github/workflows/build.yml

```diff
@@ -81,13 +81,23 @@ jobs:
       run: |
         if [ ! -e $PREFIX/bin/bundle ]; then
           export PATH="$PREFIX/bin:$PATH"
-          gem install bundler -v '~> 1' --no-document
```

Comment:
If you wanted, we can alternatively just remove `--no-document` and don't branch at all.